biotracks/readfile.py

```diff
@@ -5,28 +5,12 @@
 
 import pandas as pd
 import xlrd
-from xlrd import XLRDError
 
 from .names import (
     X_COORD_NAME, Y_COORD_NAME, FRAME_NAME, OBJECT_NAME, LINK_NAME
 )
 
 
-def xls_to_csv(xls_file):
-    """Utility function to read Excel files."""
-
-    x = xlrd.open_workbook(xls_file)
-    x1 = x.sheet_by_index(0)
-    name, extension = os.path.splitext(xls_file)
-
-    with open(name + '.csv', 'w', encoding='utf-8') as csv_file:
-        writecsv = csv.writer(csv_file, quoting=csv.QUOTE_NONE)
-        for rownum in range(x1.nrows):
-            writecsv.writerow(x1.row_values(rownum))
-
-    csv_file.close()
-
-
```

Comment:
We can add (perhaps in a future PR) a generic Excel reader that behaves like the others, i.e., reads an Excel file (this will need a config objects since it's generic) and returns objects/links data structures. `xls_to_csv` was also writing to CSV, but maybe it's better to decouple writing code to a different module.